level_editor.py

```diff
@@ -0,0 +1,130 @@
+"""
```

Comment:
i know this is spaghetti af, dont worry about it too much since none of this runs with the game, and doesnt have to be in the release package. 

also definetly not done